### ISSUE TITLE
NECB 2011 and 2015 HRV Non-Compliance

### ISSUE THREAD

**Issue by padmassun** [canmet-energy/btap_tasks#65]
## HRV Compliance
NECB 2011: 5.2.10.1
NECB 2015: 5.2.10.1

NECB code regarding MURBS (§`5.2.10.4 Heat recovery in Dwelling Units`) has _**not**_ been implemented in _**both**_ NECB 2011 and 2015
>5.2.10.4.(2) Heat recovery ventilators used to meet the requirements of Sentence (1) shall have a sensible-heat-recovery efficiency, when tested in conformance with the low-temperature thermal and ventilation test methods described in CAN/CSA-C439, "Rating the Performance of Heat/Energy-Recovery Ventilators," of
             a) at least 65% at an outside air test temperature of 0°C, and
             b) not less than that required by Table 5.2.10.4. for the 2.5% January design temperature for the *building*'s location, as listed in Appendix C of Division B of the NBC.

**<p align="center">Table 5.2.10.4 <br> Performance of heat-recovery Ventilators <br> Forming part of sentence 5.2.10.4.(2)</p>**

2.5% January Design Temperature at Building Location | Outside Air Test temperature at Station 1, °C | Sensible-Heat-Recovery Efficiency, %
-- | -- | --
\>= -10 | 0 | 65
< -10 and > -30 | -25 | 55
<= -30 | -40 | 45

**Comment by padmassun** [comment]
Related to https://github.com/canmet-energy/btap_tasks/issues/48

**Comment by mlubun** [comment]
This has been implemented correctly, as HRV's are only required in dwelling units with dedicated outdoor air units (ie in-suite ventilation).  We use NECB Sys-1 which is a dedicated outdoor air unit, but we apply it as a central dedicated outdoor air unit (MUA unit) with dwelling unit PTACs (correct implementation of NECB 2011 and 2015).  See attached clarification
[NECB 2011 and 2015 hrv.docx](https://github.com/canmet-energy/btap_tasks/files/2220943/NECB.2011.and.2015.hrv.docx)

**Comment by padmassun** [comment]
I will be implementing the check in QAQC only. I will not be making any changes to the standards.

**Comment by padmassun** [comment]
Depending on #124

**Comment by padmassun** [comment]
The logic and the code has been documented in #124, _**but has not been merged into nrcan**_.There were issues writing a proper test, because of how the current NECB standards code is written. (i.e. Spaces with similar spacetypes that has the same schedules being grouped together within a single AirLoop.) 

This makes it difficult to test an external model that has multiple dwelling zones each which has its own thermal zone, served by a Airloop per dwelling unit.
